src/components/Sum.jsx

```diff
@@ -1 +1,19 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => (
+  <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
+
+// eslint-disable-next-line
+Sum.PropTypes = {
```

Comment:
you will get this message in console `index.js:1 Warning: Component Sum declared `PropTypes` instead of `propTypes`. Did you misspell the property assignment?` it should be `Sum.propTypes`